### PR TITLE
[Test]: Update .gitignore to avoid stage errors

### PR DESCRIPTION
**Pull request by skatlapa**
This fixes submodule mapping errors while git cloning

### CODE REVIEW

**Comment by skatlapa** [comment]
Basically, this aims to fix this error:
 
Cloning "https://github.com/CentOS-PaaS-SIG/contra-env-setup.git" ...
error: build error: No submodule mapping found in .gitmodules for path 'centos7-vagrant'

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+.venv
+.workspaces
+/.eggs
+/AUTHORS
+/ChangeLog
+/ansible.cfg
+/build
+/dist
 # IntelliJ project files
-.idea
```
Comment (arilivigni) [inline .gitignore]: We want to ignore .idea and .pyc too

File: .gitignore [hunk 2]
```diff
@@ -1,10 +1,25 @@
 # Created by .ignore support plugin (hsz.mobi)
-### Example user template
-
+### Needed for contra-env-setup repo
+#(skatlapa)TODO: remove if needed, added extra for cushioning
+*.egg-info
+*.eggs
+*.log
+*.pyc
```
Comment (skatlapa) [inline .gitignore]: @arilivigni they are here :) Should've pointed in the comments, my bad.

**Comment by skatlapa** [comment]
If this still fails - for submodules use we need "git clone --recursive" so we can do that manually while cloning to w/a the error but for the CI, I believe the ghprb plugin does not support "--recursive" option atm.. @joejstuart would you have any idea about this?

**Comment by skatlapa** [comment]
just keeping @ssegalredhat in loop..

**Comment by skatlapa** [comment]
Merging this, I will update the prepare environment stage in another PR shortly to address the CI for "--recursive"

**Pull request merged**